src/httprpc.cpp

```diff
@@ -13,13 +13,21 @@
 #include <util/strencodings.h>
 #include <util/system.h>
 #include <util/translation.h>
+#include <boost/algorithm/string.hpp>
 #include <walletinitinterface.h>
 
 #include <memory>
+#include <vector>
+#include <map>
+#include <string>
+#include <algorithm>
 #include <stdio.h>
 
 #include <boost/algorithm/string.hpp> // boost::trim
 
+std::map<std::string, std::vector<std::string>> rpcWhitelist;
```

Comment:
I think using unordered_set would be better than vector.

Reply:
I don't think so, vectors are faster for linear access which is required.

Reply:
may be better to have `map<string, set<string>>`. Because these are fixed size small sets it will be pretty fast to look up. The `set<string>` also has the benefit of de-duplicating parameters.